PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -450,11 +449,7 @@ void PsychSetTextBGColor(int context, double* color)
 // renderbackend needs a different geometric setup:
 void PsychSetTextViewPort(int context, double xs, double ys, double w, double h)
```

Comment:
Well, the double _vxs, _vys, _vw, _vh variables are no longer used anywhere (the normal PTB projection matrix is fine, and no messing with vertical positions is needed anymore), so no need to keep them around. I considered removing the whole PsychSetTextViewPort call completely, but realized it is a useful interface to have for possible later plugins. So just keep it, but let it do nothing is what i thought.